process.on('uncaughtException', (err) => {
      console.log(`ğŸ›‘ | Une erreur de type Execption est survenue: ${err}`);
});

process.on('unhandledRejection', (err) => {
      console.log(`ğŸ›‘ | Une erreur de type Rejection est survenue: ${err}`);
});